### PR TITLE
chore: fix deepscan issues

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

fix deepscan issues
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

n/a

</details>

### CODE REVIEW

File: websites/B/BBC/presence.ts [hunk 1]
```diff
@@ -234,7 +234,7 @@ presence.on("UpdateData", async () => {
 
 		if (path.includes("/play/")) {
 			const isLive = path.includes("live:");
-			setCover(soundData.programmes.current.image_url);
+			setCover(soundData.programmes.current.imageUrl);
```
Comment (Bas950) [inline websites/B/BBC/presence.ts:237]: yea pretty sure that won't work, as it is deta they get out of a pegeletiable

File: websites/B/BBC/presence.ts [hunk 2]
```diff
@@ -791,7 +790,7 @@ interface MediaData {
 interface SoundData {
 	programmes: {
 		current: {
-			image_url: string;
+			imageUrl: string;
```
Comment (Bas950) [inline websites/B/BBC/presence.ts:793]: ditto

**Review by Bas950 — CHANGES_REQUESTED** [review]
all ditto

**Comment by DarkVillager** [comment]
Issue is already fixed.

**Pull request closed without merging**